### ISSUE TITLE
UserApi: Unable to GET list of users

### ISSUE THREAD

**Issue by kornys** [EnMasseProject/enmasse#1577]
I'm not able to get list of users.

```
23:17:59,315 [main] INFO io.enmasse.systemtest.apiclients.UserApiClient - GET-user: path '/apis/user.enmasse.io/v1alpha1/namespaces/myproject/messagingusers'
23:18:00,613 [main] INFO io.enmasse.systemtest.bases.TestBase - {"apiVersion":"user.enmasse.io/v1alpha1","kind":"MessagingUserList","items":[]}
```
myproject is project where enmasse is deployed
same result I have got when instead of myproject using addressspace name.

**Comment by kornys** [comment]
https://github.com/EnMasseProject/enmasse/pull/1563

**Comment by kornys** [comment]
@lulf so Ive fixed it, issue was in keycloak-controller. I pushed fix into branch #1563, see https://github.com/EnMasseProject/enmasse/pull/1563/commits/45d5e9468cccfe24b93a795c2ac194cfa17486a0
